### PR TITLE
ACS-8139 Expect agent id from ADW and pass it to HxI.

### PR DESCRIPTION
**Pull request by tpage-alfresco**

### CODE REVIEW

File: hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/rest/api/model/QuestionModel.java [hunk 1]
```diff
@@ -52,6 +52,8 @@ public class QuestionModel
     @NotBlank
     private String question;
     @NotBlank
+    private String agentId;
+    @NotBlank
```
Comment (mpichura) [inline hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/rest/api/model/QuestionModel.java:56]: Will those validation annotations work in JAMP?

File: hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/model/Question.java [hunk 2]
```diff
@@ -38,5 +38,7 @@ public class Question
     @NotBlank
     private final String question;
     @NotBlank
+    private final String agentId;
+    @NotBlank
```
Comment (mpichura) [inline hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/model/Question.java:42]: Will those validation annotations work in JAMP?

**Review by Pawel-608 — APPROVED** [review]
LGTM

File: hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/rest/api/model/QuestionModel.java (hunk 1 above)
Comment (tpage-alfresco) [reply hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/rest/api/model/QuestionModel.java:56]: No - I raised a second PR immediately afterwards to take them out again!

**Pull request merged**